src/diffusers/schedulers/scheduling_pndm_flax.py

```diff
@@ -374,18 +383,20 @@ def step_plms(
             `tuple`. When returning a tuple, the first element is the sample tensor.
 
         """
+
         if state.num_inference_steps is None:
             raise ValueError(
                 "Number of inference steps is 'None', you need to run 'set_timesteps' after creating the scheduler"
             )
 
-        if not self.config.skip_prk_steps and len(state.ets) < 3:
-            raise ValueError(
-                f"{self.__class__} can only be run AFTER scheduler has been run "
-                "in 'prk' mode for at least 12 iterations "
-                "See: https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/pipeline_pndm.py "
-                "for more information."
-            )
+        # NOTE: There is no way to check in the jitted runtime if the prk mode was ran before
+        # if not self.config.skip_prk_steps and state.counter < 3:
+        #     raise ValueError(
+        #         f"{self.__class__} can only be run AFTER scheduler has been run "
+        #         "in 'prk' mode for at least 12 iterations "
+        #         "See: https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/pipeline_pndm.py "
+        #         "for more information."
+        #     )
```

Comment:
You are right, feel free to remove the commented code if you want (maybe just leave the note?)

```suggestion
        # NOTE: There is no way to check in the jitted runtime if the prk mode was ran before
```